docs/docs/cmd/spfx/project/project-github-workflow-add.mdx

```diff
@@ -0,0 +1,83 @@
+import Global from '/docs/cmd/_global.mdx';
+
+# spfx project github workflow add
+
+Adds a GitHub workflow for a SharePoint Framework project
+
+## Usage
+
+```sh
+m365 spfx project github workflow add [options]
+```
+
+## Options
+
+```md definition-list
+`-n, --name [name]`
+: Name of the workflow that will be created. If none is specified a default name will be used 'Deploy Solution ${name of sppkg file}'
+
+`-b, --branchName [branchName]`
+: Specify the branch name which should trigger the workflow on push. If none is specified a default will be used which is 'main'
+
+`-m, --manuallyTrigger`
+: When specified a manual trigger option will be added to the workflow: `workflow_dispatch`
+
+`-l, --loginMethod [loginMethod]`
+: Specify the login method used for the login action. Possible options are: `user`, `application`. Default `application`'
+
+`-s, --scope [scope]`
+: Scope of the app catalog: `tenant`, `sitecollection`. Default is `tenant`
+
+`-u, --siteUrl [siteUrl]`
+: The URL of the site collection where the solution package will be added. Required if scope is set to `sitecollection`
+
+`--skipFeatureDeployment`
+: When specified and the app supports tenant-wide deployment, deploy it to the whole tenant
+
+`--overwrite`
+: Set to overwrite the existing package file.
```

Comment:
Let's clarify that this applies to the workflow overriding the .sppkg. Right now it seems like a copy-paste error

Reply:
overriding oooor overwriting 😉

Reply:
✅